addons/project/models/project.py

```diff
@@ -121,8 +121,8 @@ def _compute_task_count(self):
         result_wo_subtask = defaultdict(int)
         result_with_subtasks = defaultdict(int)
         for data in task_data:
-            result_wo_subtask[data['project_id'][0]] += data['display_project_id']
-            result_with_subtasks[data['project_id'][0]] += data['project_id_count']
+            result_wo_subtask[(data['project_id'] or [False])[0]] += data['display_project_id']
+            result_with_subtasks[(data['project_id'] or [False])[0]] += data['project_id_count']
```

Comment:
Hello @nseinlet 🙂,

I would rather add ('project_id', 'in', self.ids) in the read_group domain, don't you think? @tlibioulle confirmed it has been removed by mistake.

Have a great day!

Reply:
Of course, do the right fix, this was just a crappy workaround.

Reply:
We have identified that this could have happened from either the timesheet app in the kanban and list view. We will check with the PO if that behavior is really wanted or not.

Thanks @nseinlet for having brought this to us 🙂